src/bundle/Resources/config/services.yml

```
@@ -390,3 +390,17 @@ services:
         parent: hautelook.router.template
         calls:
             - [ setOption, [ strict_requirements, ~ ] ]
+
+    Ibexa\Bundle\Rest\Routing\ExpressionLanguage\ContentTypeHeaderMatcherFactory:
+        arguments:
+            $mediaTypeParser: '@Ibexa\Contracts\Rest\Input\MediaTypeParserInterface'
+
+    ibexa.routing.expression_language.get_media_type:
```

rather 
```suggestion
    ibexa.rest.expression_language.get_media_type:
```
Even though we're using this for routing, AFAIR expression language can be applied in more places.